src/Symfony/Component/VarDumper/Tests/Caster/IntlCasterTest.php

```diff
@@ -147,6 +147,44 @@ public function testCastNumberFormatter()
     MONETARY_GROUPING_SEPARATOR_SYMBOL: "$expectedSymbol18"
   }
 }
+EOTXT;
+        $this->assertDumpEquals($expected, $var);
+    }
+
+    public function testCastIntlTimeZoneWithDST()
+    {
+        $var = \IntlTimeZone::createTimeZone('Europe/Berlin');
```

Comment:
Shouldn't we take another TZ, since we know Europe is going to give up DST next year?

Reply:
Yes sure, I totally forgot about that.